includes/amp-helper-functions.php

```diff
@@ -512,7 +512,7 @@ function amp_filter_script_loader_tag( $tag, $handle ) {
  * @param string $href   Link URL.
  * @return string Link tag HTML.
  */
-function amp_filter_font_style_loader_tag_with_crossorigin_anonymous( $tag, $handle, $href ) {
+function amp_filter_font_style_loader_tag_with_crossorigin_anonymous( $tag, $handle, $href = '' ) {
```

Comment:
Also, if we went with this, more would be needed as an empty `$href` would mean that the URL would need to be extracted from the `$tag` via regular expression.

Reply:
Yes. Concatenated CSS is being skipped currently because of not having `$href`.

But I believe we will have to add this regex to get `href` from `$tag`, if we have to go with this solution.

Reply:
That's right. The `$href` would need to be extracted from the `$tag` if it is empty. For example:

```php
if ( empty( $href ) && preg_match( '#<link.+?href="(.+?)"#', $tag, $matches ) ) {
    $href = html_entity_decode( $tag[1], ENT_QUOTES );
}
```

Ultimately this would be a fix for themes and plugins which are incorrectly applying the `style_loader_tag` filter.

Reply:
@westonruter 
Made small changes to allow single or double quotes in regex.
```php 
if ( empty( $href ) && preg_match( '#<link.+?href=[\'"](.+?)[\'"]#', $tag, $matches ) ) {
    $href = html_entity_decode( $matches[1], ENT_QUOTES );
}
```

Since we get tag like this:
```
<link rel='stylesheet' id='all-css-0' href='https://amp-wp.test/_static/??/wp-includes/css/dashicons.css,/wp-includes/css/admin-bar.css?m=1534398924j' type='text/css' media='all' /> 
```